ui/app/routes/workspace/projects/project/settings/config-variables.ts

```diff
@@ -0,0 +1,26 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+import ApiService from 'waypoint/services/api';
+import { ConfigGetRequest, ConfigGetResponse, Ref } from 'waypoint-pb';
+import { Params as ProjectRouteParams } from 'waypoint/routes/workspace/projects/project';
+export default class WorkspaceProjectsProjectSettingsConfigVariables extends Route {
+  @service api!: ApiService;
+
+  async model(): Promise<ConfigGetResponse.AsObject> {
+    let ref = new Ref.Project();
+    let params = this.paramsFor('workspace.projects.project') as ProjectRouteParams;
+    ref.setProject(params.project_id);
+    let req = new ConfigGetRequest();
+    req.setProject(ref);
+
+    let config = await this.api.client.getConfig(req, this.api.WithMeta());
+    return config?.toObject();
+  }
+
+  setupController(controller, model, transition) {
+    super.setupController(controller, model, transition);
+    let project = this.modelFor('workspace.projects.project');
+
+    controller.project = project;
```

Comment:
Is it safe to do this without `set`, given `controller.project` isn’t a `@tracked` property? I would _guess_ it’s OK but I’m not 100% sure.

Reply:
good question 🤔. I'll verify and update tomorrow

Reply:
looked into it: it's fine and the route updates the controller properly every time. The only concern would be if we were interacting with this controller from outside the route, but we're not.